src/main/java/bit/javaoop/Company.java

```diff
@@ -0,0 +1,54 @@
+package bit.javaoop;
+
+public class Company {
+    public static final int companyEmployees = 2;
+    public static final int companyManagers = 3;
```

Comment:
- Hardcode'owanie takich rzeczy jest dosyć złym pomysłem bardzo nas ogranicza, lepiejbyłoby przyjąć takie parametry w konstruktorze
- Poza tym zamiast trzymania takich dziwsnych parametrów powinna być poprostu metoda w tym stylu
```
public int getCompanyEmployeesNumber(){
    return this.employeesList.size();
}
```
- A jak już robisz takie stałe to konwencja nazewnictwa stałych JEST_TAKA
- I generalnie raczej powinny być prywatne